### ISSUE TITLE
[BUG] gas bottle stuck to hand + don't know how to fill

### ISSUE THREAD

**Issue by TIKIRobo** [tsunamayo/Starship-EVO#5290]
### Short Description of the issue:
after holding gas bottle in hand it will remain stuck to the players hand even after they delete the bottle out of their inventory, if this is intentional maybe make it stuck to the waist instead? also, how do you fill up these gas bottles? im having a hard time actually using them/getting full ones like you have in your youtube video 
...

### Steps to reproduce the issue:
craft gas bottle
pick it up
switch to an empty hotbar slot
...

### Environment
- Starship EVO version: 23w38a

### Additional information:

https://github.com/tsunamayo/Starship-EVO/assets/56371294/aa18e4c2-78fb-4328-897c-e60e2be81ca5

**Comment by tsunamayo** [comment]
@TIKIRobo Sorry I forgot to add a HUD message that says how to use it: press the Rotate Key on a tank to fill the bottle. I will check for the bug.

**Comment by TIKIRobo** [comment]
oh okay thank you!

**Comment by ProPeach** [comment]
> @TIKIRobo Sorry I forgot to add a HUD message that says how to use it: press the Rotate Key on a tank to fill the bottle. I will check for the bug.

Makes sense! Will there be an automatic way to fill the bottles? Just thinking about the process of coming back to your ship to refill your tanks - opening up your inventory, dragging them down to your bar, finding a tank to interact with then doing the reverse might not be a very comfortable thing to do frequently. 


For example, I think Space Engineers removes this tedium by recharging/refilling the suit when you sit down and pilot a ship. Maybe the suit could automatically pull gas from the atmosphere in your ship to refill your personal tanks?
Loving all this progress to survival!

**Comment by tsunamayo** [comment]
@ProPeach So refueling will be auto at npc station, at your station you will need to dock, but of course the best way will be to harvest nearby the star.
For personal bottle yes I want to do an autofill for o2 when you are inside an atmosphere/ship. I will add an inventory slot so you can swap the bottle if needed for long exploration trip.
I will add a bottle connector to generate power from a bottle for small ship / hovercraft, so you can quickly swap those for refuel (so you dont need to dock or manual fill a tank). I will add different bottle format too.

**Comment by ProPeach** [comment]
The dedicated inventory slot was very similar to what I had in mind too, sounds great!
The Bottle Connector is an interesting one, so it'll produce power directly without the need for a Reactor if I'm reading that right? I suppose it might be available earlier on in the game before reactors. Definitely sounds more convenient for small hoverbikes etc so you don't need to line up a docking port perfectly

**Comment by Crimson-Artist** [comment]
> @ProPeach So refueling will be auto at npc station, at your station you will need to dock, but of course the best way will be to harvest nearby the star. For personal bottle yes I want to do an autofill for o2 when you are inside an atmosphere/ship. I will add an inventory slot so you can swap the bottle if needed for long exploration trip. I will add a bottle connector to generate power from a bottle for small ship / hovercraft, so you can quickly swap those for refuel (so you dont need to dock or manual fill a tank). I will add different bottle format too.

if we will refuel while docked then does that mean that hovercraft will gain the ability to dock to shipyards? currently they can not
